types/jquery.pnotify/index.d.ts

```diff
@@ -338,7 +338,7 @@ interface PNotifyConstructor {
     /**
      * Reposition the notices, optionally animating their movement.
      */
-    positionAll(animate: string): void
+    positionAll(animate?: string): void
```

Comment:
One last thing, if you look at the examples I've shown previously:

https://github.com/sciactive/pnotify/blob/ffb47b2b4f1ed4aaf6c60db0b84d6218a05dfd22/src/pnotify.js#L56
https://github.com/sciactive/pnotify/blob/ffb47b2b4f1ed4aaf6c60db0b84d6218a05dfd22/src/pnotify.js#L204

Then you'll see the animate parameter is a optional **boolean** instead of a string, or am I missing something?

Reply:
I've checked the source some more. It's a little confusing because this value is assigned to the animation property of the PNotifyStack. In the stack it's a boolean, so it's not like the animation setting which expects a string. Nice and confusing :smiley: 

positionAll(animate?: boolean): void

Reply:
you are correct it should be boolean 
positionAll is called by queuePosition which takes animate as a boolean